packages/EasyPagination/src/Paginators/DoctrineDbalLengthAwarePaginator.php

```diff
@@ -64,7 +64,16 @@ protected function doGetResult($queryBuilder): array
      */
     protected function doGetTotalItems($queryBuilder, string $countAlias): int
     {
-        $result = (array)$this->conn->fetchAssoc($queryBuilder->getSQL(), $queryBuilder->getParameters());
+        // @TODO remove this method_exists check when doctrine/dbal is updated to a version containing the fetchAssociative method (>=2.11)
+        if (\method_exists($this->conn, 'fetchAssociative')) {
+            $result = (array)$this->conn->fetchAssociative(
+                $queryBuilder->getSQL(),
+                $queryBuilder->getParameters(),
+                $queryBuilder->getParameterTypes()
+            );
+        } else {
```

Comment:
Pls avoid `else` usages if possible like it's described in code conventions.

Reply:
Updated